package.json

```diff
@@ -9,27 +9,29 @@
   "dependencies": {
     "dexie": "^3.0.3",
     "dragula": "^3.7.3",
+    "electron-installer-debian": "^3.1.0",
+    "electron-installer-redhat": "^3.2.0",
```

Comment:
These should only be listed in `optionalDependencies` I think, I'm not sure why npm is moving them (although it did the same thing to me last time I installed).

Reply:
Got it.